doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -5,180 +5,182 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
+=== Dashbuilder Runtime
 
-* After creating dashboards using datasets and pages, you can export these dashboards from {CENTRAL} using *custom export*.
-* You can import, access, and run the dashboards that are exported from {CENTRAL} in dashbuilder runtime.
-* Dashbuilder runtime accesses the {KIE_SERVER} REST API to run the queries from imported {KIE_SERVER} datasets.
-* Dashbuilder runtime provides embedded capabilities that allow you to set a specific dashboard from an instance of dashbuilder runtime on your web applications.
-* In Dashbuilder runtime, you can import multiple dashboards in a single dashbuilder runtime instance.
-* You can create and add external components that can be a part of the dashboard page.
+The following list provides a summary of Dashbuilder Runtime updates:
 
-== Entando AppBuilder
-Entando AppBuilder is now officially removed from the {PRODUCT} 7.9. {PRODUCT} no longer includes entitlements for Entando AppBuilder. The feature references and documentation is not be available from the 7.9 release.
+* You can automatically deploy the dashboards from {CENTRAL} on Dashbuilder Runtime. {CENTRAL} is linked to Dashbuilder Runtime using a gradual export feature. You can select datasets and pages, but instead of downloading a `ZIP` file, you can click the *Open* button. After clicking the *Open* button, the selected data is exported, and Dashbuilder Runtime updates the model content.
+You can use `dashbuilder.runtime.multi`, `dashbuilder.runtime.location`, and `dashbuilder.export.dir` system properties to enable this feature.
++
+For more information, see {URL_MANAGING_SETTINGS}#assembly-configuring-central[_{CONFIGURING_CENTRAL}_].
+
+* Dashboards that are imported in Dashbuilder Runtime contain a default page. The following are the updates of the Dashbuilder Runtime default page:
+
+** If an imported dashboard consists of only one page, then the page is used as the default page.
+** If a page is named as `index`, then the index page is used as the default page.
+** In other cases, the generic home page of the Dashbuilder Runtime is used as the default page.
+
+* Dashbuilder Runtime is now available with a new heatmap component feature. You can add the heatmap component to the pages and export it to the Dashbuilder Runtime. Heatmap components are used to display heat information over a process diagram, and you can create, edit, and build a dashboard using the heatmap component. The heat information is retrieved from {KIE_SERVER} datasets.
++
+For more information, see {URL_MANAGING_SETTINGS}#assembly-building-custom-dashboard-widgets[_{BUILDING_WIDGETS}_].
 
 endif::PAM[]
 
 ifdef::PAM[]
 
-== Process Designer
+== New standalone BPMN and DMN editors
```

Comment:
Shouldn't the standalone DMN editor be described for RHDM as well?

Reply:
@krisv, I received the feedback from Martin Weiler that **New standalone BPMN and DMN editors** feature not applicable to RHDM so I removed from RHDM doc. In that case, I will include it in the RHDM doc as well. 
Can you please confirm?

Reply:
@kaldesai afaik Martin was wondering whether the BPMN editor should be mentioned for RHDM (I think there was already agreement that at least the DMN editor should be mentioned).  But BPMN is also used for RuleFlow in RHDM, so I don't think there's an issue with mentioning it in RHDM as well, and Ivo confirmed in his JIRA comment as well.

Reply:
@krisv, Okay thank you for the confirmation I will include it in the RHDM doc.